scripts/image_build.sh

```diff
@@ -114,8 +114,9 @@ echo 'get_ipython().magic(u"%reload_ext sqlflow.magic")' >> $IPYTHON_STARTUP/00-
 echo 'get_ipython().magic(u"%autoreload 2")' >> $IPYTHON_STARTUP/00-first.py
 curl https://raw.githubusercontent.com/sql-machine-learning/sqlflow/develop/example/jupyter/example.ipynb --output /workspace/example.ipynb
 
-# 9. install xgboost-launcher
-pip install xgboost-launcher==0.0.4
+# Re-enable this after Ant-XGBoost is ready.
+# # 9. install xgboost-launcher
+# pip install xgboost-launcher==0.0.4
```

Comment:
The `xgboost-laucnher` included the `dmlc/xgboost` package, if we want to skip ant-xgboost unit test, maybe we also need to install `xgboost==0.90` to pass the `dmlc/xgboost` unit test.

Reply:
Added. Thanks for the reminder!